test/e2e-tests-yaml.sh

```diff
@@ -1,55 +0,0 @@
-#!/usr/bin/env bash
-
-# Copyright 2019 The Tekton Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-# This script calls out to scripts in tektoncd/plumbing to setup a cluster
-# and deploy Tekton Pipelines to it for running integration tests.
-
-source $(git rev-parse --show-toplevel)/test/e2e-common.sh
-
-# Script entry point.
-
-initialize $@
-
-header "Setting up environment"
-
-# Handle failures ourselves, so we can dump useful info.
-set +o errexit
-set +o pipefail
-
-install_pipeline_crd
```

Comment:
do we need this call to install_pipeline_crd before running our tests or was it unnecessary?

Reply:
`install_pipeline_crd` is called by `test/e2e-tests.sh`, which later invokes the example tests, so for the Github CI all is well.

For local instances, I'd assumed the `ko apply -f ./config/` suggested at the beginning of [`test/README.md`](https://github.com/tektoncd/pipeline/tree/master/test#tests) would have already been run, so there wouldn't be a need for a subsequent `install_pipeline_crd`. 

Is this ok?